components/brave_federated_learning/data_stores/data_store.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_DATA_STORES_DATA_STORE_H_
+#define BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_DATA_STORES_DATA_STORE_H_
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include "base/files/file_path.h"
+#include "base/memory/ref_counted.h"
+#include "sql/database.h"
+#include "sql/meta_table.h"
```

Comment:
can we move this to the implementation, doesn't seem to be needed in header

Reply:
I'm going to remove it, not needed in the implementation either